packages/broker/src/plugins/storage/StorageConfig.ts

```diff
@@ -1,38 +1,19 @@
 import fetch from 'node-fetch'
-import { Logger } from 'streamr-network'
-import { StreamPart } from '../../types'
-import { StreamMessage, keyToArrayIndex } from 'streamr-network/dist/streamr-protocol'
+import { Logger, Protocol } from 'streamr-network'
+import { SPID } from 'streamr-network/dist/streamr-protocol'
 import { SubscriptionManager } from '../../SubscriptionManager'
 
 const logger = new Logger(module)
 
-type StreamKey = string
-
 export interface StorageConfigListener {
-    onStreamAdded: (streamPart: StreamPart) => void
-    onStreamRemoved: (streamPart: StreamPart) => void
-}
-
-/*
- * Connects to Core API and queries the configuration there.
- * Refreshes the config at regular intervals.
- */
-const getStreamFromKey = (key: StreamKey): StreamPart => {
-    const [id, partitionStr] = key.split('::')
-    return {
-        id,
-        partition: Number(partitionStr)
-    }
-}
-
-const getKeyFromStream = (streamId: string, streamPartition: number): StreamKey => {
-    return `${streamId}::${streamPartition}`
+    onSPIDAdded: (spid: Protocol.SPID) => void
+    onSPIDRemoved: (spid: Protocol.SPID) => void
```

Comment:
I'm wonder if it would be more clear to refer to the stream partitions by full name?
```suggestion
    onStreamPartitionAdded: (spid: Protocol.SPID) => void
    onStreamPartitionRemoved: (spid: Protocol.SPID) => void
```